rest/route/service.go

```diff
@@ -122,6 +122,7 @@ func AttachHandler(app *gimlet.APIApp, opts HandlerOpts) {
 	app.AddRoute("/patches/{patch_id}/abort").Version(2).Post().Wrap(checkUser, submitPatches).RouteHandler(makeAbortPatch(sc))
 	app.AddRoute("/patches/{patch_id}/restart").Version(2).Post().Wrap(checkUser, submitPatches).RouteHandler(makeRestartPatch(sc))
 	app.AddRoute("/projects").Version(2).Get().Wrap(checkUser).RouteHandler(makeFetchProjectsRoute(sc))
+	app.AddRoute("/projects/test_alias").Version(2).Get().Wrap(checkUser).RouteHandler(makeTestProjectAliasHandler(sc))
```

Comment:
If you choose to make version required, then this makes more sense as /versions/{version_id}

Reply:
It's required for now, but I think of this more as "check for this alias in this project" and version is just the most efficient way to get the project config

Reply:
Hmm reasonable. Will it be too much to ask users to include the project ID when they use this route so in the future if we choose to not make version required the route will still make sense (as it'll get the most recent config for that project)? Otherwise for that use case we can create a new route that just calls the same handler but.

Reply:
I do still think it's cleaner to make version not required, I could see not-cloud wanting to use this, or cloud changing their mind about the use case.